core/app/models/spree/order_cancellations.rb

```diff
@@ -29,7 +29,7 @@ def short_ship(inventory_units, whodunnit:nil)
         end
 
         update_shipped_shipments(inventory_units)
-        Spree::OrderMailer.inventory_cancellation_email(@order, inventory_units).deliver if Spree::OrderCancellations.send_cancellation_mailer
+        Spree::OrderMailer.inventory_cancellation_email(@order, inventory_units.to_a).deliver_later if Spree::OrderCancellations.send_cancellation_mailer
```

Comment:
:+1: So without the `to_a` we get a serialization because it's an `AR::Relation`? Neat.